src/llama.cpp

```diff
@@ -19601,6 +19829,25 @@ static int32_t llama_chat_apply_template_internal(
         if (add_ass) {
             ss << "<|start_header_id|>assistant<|end_header_id|>\n\n";
         }
+    } else if (tmpl.find("chatglm3") != std::string::npos || tmpl.find("[gMASK]sop") != std::string::npos) {
```

Comment:
Same here: after merging with latest master, you can do:

```suggestion
    } else if (tmpl == "chatglm3" || tmpl_contains("[gMASK]sop")) {
```